bool start
---
int8 err
